compiler/rustc_codegen_gcc/gcc_path

```diff
@@ -0,0 +1 @@
+/home/bouanto/Ordinateur/Programmation/Projets/gcc-build/build/gcc
```

Comment:
What happened here?

Reply:
Oops. My bad. Is there a good way to ignore changes to a commited file in git? I used `git update-index --assume-unchanged gcc_path` but I keep having issues.